lib/internal/process/main_thread_only.js

```diff
@@ -154,6 +154,14 @@ function setupSignalHandlers(internalBinding) {
       delete signalWraps[type];
     }
   });
+
+  // re-arm pre-existing signal event registrations
+  // with this signal wrap capabilities.
+  const events = process.eventNames();
+  events.forEach((ev) => {
```

Comment:
minor nit: we can just do `process.eventNames().forEach((ev) => {`

Reply:
@mscdex - done, and squashed. thanks.